discovery-provider/src/queries/queries.py

```diff
@@ -1872,3 +1872,146 @@ def get_top_followee_saves(type):
                     track['user'] = user
 
     return api_helpers.success_response(tracks)
+
+# Retrieves the top users for a requested genre under the follow parameters
+# - A given user can only be associated w/ one genre
+# - The user's associated genre is calculated by tallying the genre of the tracks and taking the max 
+#   - If there is a tie for # of tracks in a genre, then the first genre alphabetically is taken 
+# - The users associated w/ the requested genre are then sorted by follower count
+# Route Parameters
+#   urlParam: {Array<string>?}  genre       List of genres to query for the 'top' users
+#   urlParam: {boolean?}        with_user   Boolean if the response should be the user ID or user metadata defaults to false
+@bp.route("/top_genre_users", methods=("GET",))
```

Comment:
what do you think about a more restful route like `users/genre/top`?

Reply:
I was looking at some of the endpoints added by Ray in the new discovery routes and tried to follow the pattern for consistency. But, yeah I can change this route.